public/Face.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { faceToSvgString } from "../src/faceToSvgString";
+import { FaceConfig, Overrides } from "../src/types";
+import { objStringifyInOrder } from "./utils";
+import { useInView } from "react-intersection-observer";
+
+/*
+    This component is responsible for rendering the face SVG string
+    It uses the faceToSvgString function to generate the SVG string
+    It requires the faceConfig to be set, with the overrides being optional
+    The width and className are also optional
+    The SVG string is then rendered in a div element
+    The SVG string is set as the innerHTML of the div element
+*/
+
+export const Face = ({
+  faceConfig,
+  overrides,
+  maxWidth,
+  width,
+  className,
+  lazyLoad,
+}: {
+  faceConfig: FaceConfig;
+  overrides?: Overrides;
+  maxWidth?: number;
+  width?: number;
+  className?: string;
+  lazyLoad?: boolean;
+}) => {
+  const [ref, inView] = useInView({
+    triggerOnce: true,
+    threshold: 0,
+  });
+
+  const faceSvg =
+    inView || !lazyLoad ? faceToSvgString(faceConfig, overrides) : "";
```

Comment:
It's probably better to use `display` rather than `faceToSvgString` - the latter is only intended for use by the CLI for exporting an SVG without a browser. Probably `display` is much faster. Although if not it would be interesting!

Also the way you have it now, it's generating the SVG every time the Face component renders. It'd be better to not do that. For `display` you'd generally put it in a `useEffect` that runs only the first time the component is rendered (or in this case, I guess the first time it's in view).